docs/index.asciidoc

```diff
@@ -57,6 +55,74 @@ Otherwise, you are responsible for maintaining compliance.
 The Logstash open source distribution uses the MaxMind Creative Commons license
 database by default.
 
+==== Database Auto-update
+
+This plugin embedded with Creative Commons (CC) license databases.
+When Logstash works in air-gapped enviornment, CC license databases can be used indefinitely. 
+Logstash checks database update every day. It downloads the latest and can replace the old database on-the-fly.
+Once it downloads EULA license databases, it will not fallback to CC license databases.
```

Comment:
```suggestion
This plugin bundles Creative Commons (CC) license databases.
In air-gapped environments, Logstash can use CC license databases indefinitely. 
Logstash checks for database updates every day. It downloads the latest and can replace the old database while the plugin is running.
After Logstash downloads EULA license databases, it will not fallback to CC license databases.
```

Reply:
WRT "on-the-fly."  I offered a suggestion, but I'm not sure if it's correct.  Ideally, we should explain exactly how this works.